services/prometheus/prometheus.go

```diff
@@ -132,6 +134,25 @@ func (svc *Service) reload() error {
 	return errors.Errorf("%d: %s", resp.StatusCode, b)
 }
 
+func (svc *Service) loadBaseConfig() *config.Config {
+	var cfg config.Config
+
+	buf, err := ioutil.ReadFile(svc.baseConfigPath)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			svc.l.Errorf("Failed to load base prometheus config %s: %s", svc.baseConfigPath, err)
```

Comment:
Is it possible that we might not have a base file? And how important that for us? It looks like we can change level to warning.

Reply:
Yes, the base file is optional. This error message raised when the base file exists but for some reason can't be read, for example, because of wrong permissions. So when the file doesn't exist - we use the default config, if it exists - than any problem requires user's attention.

Reply:
> Is it possible that we might not have a base file?
In 99% of cases :)